backend/src/Designer/Controllers/ResourceAdminController.cs

```diff
@@ -293,10 +293,44 @@ private ValidationProblemDetails ValidateResource(ServiceResource resource, bool
                 ModelState.AddModelError($"{resource.Identifier}.rightDescription", "resourceerror.missingresourcetype");
             }
 
+            if (resource.AvailableForType == null || resource.AvailableForType.Count == 0)
+            {
+                ModelState.AddModelError($"{resource.Identifier}.availableForType", "resourceerror.missingavailablefortype");
+            }
+
             if (resource.ContactPoints == null || resource.ContactPoints.Count == 0)
             {
                 ModelState.AddModelError($"{resource.Identifier}.rightDescription", "resourceerror.missingcontactpoints");
             }
+            else
+            {
+                for (int i = 0; i < resource.ContactPoints.Count; i++)
+                {
+                    var contactItem = resource.ContactPoints[i];
+
+                    if (string.IsNullOrWhiteSpace(contactItem.Category))
```

Comment:
It would be enough that one of the values is filled out in a group. We cant expect to have all types of contact info